staging/src/k8s.io/pod-security-admission/policy/registry.go

```diff
@@ -64,26 +65,28 @@ func (r *checkRegistry) EvaluatePod(lv api.LevelVersion, podMetadata *metav1.Obj
 	if r.maxVersion.Older(lv.Version) {
 		lv.Version = r.maxVersion
 	}
-	results := []CheckResult{}
-	for _, check := range r.baselineChecks[lv.Version] {
-		results = append(results, check(podMetadata, podSpec))
-	}
+
+	var checks []CheckPodFn
 	if lv.Level == api.LevelBaseline {
-		return results
+		checks = r.baselineChecks[lv.Version]
+	} else {
+		checks = r.restrictedChecks[lv.Version]
```

Comment:
so restrictedChecks now contains *all* checks run for restricted, including non-overridden baseline checks

Reply:
correct. Added a comment to that effect.